lib/watir/exception.rb

```diff
@@ -3,13 +3,13 @@ module Exception
     class Error < StandardError; end
 
     # TODO: rename Object -> Element?
+    class WatirException < Error; end
```

Comment:
I don't see why `Watir::Exception::WatirException` is better than `Watir::Exception::Error`.